Code/Registration/include/sitkImageRegistrationMethod.h

```diff
@@ -451,19 +451,30 @@ namespace simple
     SITK_RETURN_SELF_TYPE_HEADER SetMetricMovingMask( const Image &binaryMask );
 
     /** \brief Set percentage of pixels sampled for metric evaluation.
+     *
+     * The percentage is of the number of pixels at the virtual domain
+     * per level after the shrink factor has been applied.
      *
      * The seed parameter is used to seed the pseudo-random number
      * generator used in generating the sampling set of points. If the
      * seed parameter is 0, then the wall clock is used, otherwise the
      * fixed seed is used for reproducible behavior.
      *
+     * The number of levels must either be one, or match the number of
+     * shrink factors and smoothing sigmas. If the number of
+     * percentages is one, then the same percentage is used for all
+     * levels.
+     *
```

Comment:
Not sure about the documentation above, suggested modification:

When providing multiple values the number of entries must match the number of shrink factors and smoothing sigmas. If a single value is given, then the same percentage is used for all levels.